apps/api/src/app/inbox/usecases/update-preferences/update-preferences.usecase.ts

```diff
@@ -230,18 +193,17 @@ export class UpdatePreferences {
     };
   }
 
-  private async storePreferences(item: {
+  /**
+   * Strangler pattern to migrate to V2 preferences.
+   */
+  private async storePreferencesV2(item: {
     channels: IPreferenceChannels;
     organizationId: string;
     _subscriberId: string;
     environmentId: string;
     templateId?: string;
-  }) {
+  }): Promise<void> {
     const preferences: WorkflowPreferencesPartial = {
-      all: {
-        enabled: PREFERENCE_DEFAULT_VALUE,
-        readOnly: false,
-      },
```

Comment:
There is no "all" channel preferences capability in Inbox today, so we won't persist anything.